gammapy/estimators/map/ts.py

```diff
@@ -215,7 +216,7 @@ def selection_all(self):
     def estimate_kernel(self, dataset):
         """Get the convolution kernel for the input dataset.
 
-        Convolves the model with the PSFKernel at the center of the dataset.
+        Convolves the model with the IRFs.
```

Comment:
Maybe specify: `with the IRFs at the center of the dataset (or the nearest position with non-null exposure)`.